python/whylogs/core/constraints/metric_constraints.py

```diff
@@ -17,6 +17,13 @@
 ReportResult = namedtuple("ReportResult", "name passed failed summary")
 
 
+class ConstraintsParams(TypedDict):
```

Comment:
Some of these seem redundant with the `MetricConstraint` itself?

Reply:
You're right!
In latest commit, I tried to reduce redundancy by keeping only "factory" and specific parameters of factory constraints in ConstraintsParam. The other ones: "name", "column_name", "metric", I'm getting from the MetricConstraint ( and MetricSelector) itself.